velox/exec/OutputBuffer.h

```diff
@@ -124,10 +161,36 @@ class DestinationBuffer {
   // The sequence number of the first item to pass to 'notify'.
   int64_t notifySequence_{0};
   uint64_t notifyMaxBytes_{0};
+  BufferInfo info_;
 };
 
 class Task;
 
+struct OutputBufferInfo {
```

Comment:
Please add some comments for this class, and its relationship with `BufferInfo`, whose names are very similar.  
 
Many Velox users are not Presto users, so they are not familiar with these names and may be confused.

Reply:
done.